src/components/ArtifactGreenwaveKaiState.tsx

```diff
@@ -122,7 +122,7 @@ export const FaceForGreenwaveKaiState: React.FC<
     const resultOutcome = state.gs.result?.outcome;
     const requimentType = state.gs.requirement?.type;
     // Prefer Greenwave's point of view over status from UMB message.
-    const iconName = requimentType || resultOutcome || 'unknown';
+    const iconName = resultOutcome || requimentType || 'unknown';
```

Comment:
I believe this should be 'requirementType', though we would want to do an s/requimentType/requirementType/g to grab any other typos.

Reply:
@jimbair thank you, updated.